### DISCUSSION TITLE
Cross-module dependencies

### DISCUSSION THREAD

**Discussion by reegnz** [diggerhq/digger#1323] [Q&A]
Does Digger support, (or plans to support) cross-module dependencies and ordering, like terraform enterprise (run triggers), env0 (workflows) or spacelift (stack dependencies)?

**Comment by UtpalJayNadiger** [comment]
Hey @reegnz - apologies for the delay in replying. Regarding your question above - ordering is supported in Digger, but passing IO not yet. Would love to get your thoughts on how we're thinking about it though. Would you be open to joining our slack (https://bit.ly/diggercommunity) as most of our community discussions happen there. I will close this discussion here and we can reopen it on Slack if you don't mind.
